kstreams/test_utils/test_clients.py

```diff
@@ -17,6 +17,8 @@ async def start(self):
 
 
 class TestProducer(Base, Producer):
+    __test__ = False
```

Comment:
Added these small changes to prevent pytest from trying to collect them as tests